demos/ag-alert-keyvault-advisor/azuredeploy.json

```diff
@@ -0,0 +1,64 @@
+{
+   "$schema":"https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+   "contentVersion":"1.0.0.0",
+   "parameters":{
+      "alertName":{
+         "type":"String"
+      },
+      "applicationGatewayResourceGroup":{
+         "type":"String",
+         "metadata":{
+            "description":"Resource Group example /subscriptions/subs-id/resourceGroups/rg-name"
+         }
+      },
+      "actionGroupResourceID":{
+         "type":"String",
+         "metadata":{
+            "description":"Action Group ResourceID example /subscriptions/subs-id/resourceGroups/rg-name/providers/microsoft.insights/actiongroups/action-group-name"
+         }
+      }
+   },
+   "variables":{
+      
+   },
+   "resources":[
+      {
+         "type":"microsoft.insights/activityLogAlerts",
+         "apiVersion":"2020-10-01",
+         "name":"[parameters('alertName')]",
+         "location":"Global",
+         "properties":{
+            "scopes":[
+               "[parameters('applicationGatewayResourceGroup')]"

```

Comment:
From previous published alert, continuing with App Gateway's Resource ID itself should be fine. Don't think we need to change to Resource group ID for this one as the scope identical to the other alert should work.